### PR TITLE
Add the Nordic kb layout

### PR DESCRIPTION
**Pull request by ananace**

### CODE REVIEW

File: data/matrix_layouts/razerdefault22.json [hunk 1]
```diff
@@ -267,5 +267,141 @@
...
+            {"label": "A", "matrix": [3, 2]},
+            {"label": "S", "matrix": [3, 3]},
+            {"label": "D", "matrix": [3, 4]},
+            {"label": "F", "matrix": [3, 5]},
+            {"label": "G", "matrix": [3, 6]},
+            {"label": "H", "matrix": [3, 7]},
+            {"label": "J", "matrix": [3, 8]},
+            {"label": "K", "matrix": [3, 9]},
+            {"label": "L", "matrix": [3, 10]},
+            {"label": "\u00d6", "matrix": [3, 11]},
```
Comment (z3ntu) [inline data/matrix_layouts/razerdefault22.json]: Not a Ø (`\u00d8`)?

File: data/matrix_layouts/razerdefault22.json [hunk 2]
```diff
@@ -267,5 +267,141 @@
...
+            {"label": "\u00c4", "matrix": [3, 12]},
```
Comment (z3ntu) [inline data/matrix_layouts/razerdefault22.json]: Not a Æ (`\u00c6`)?

File: data/matrix_layouts/razerdefault22.json [hunk 3]
```diff
@@ -267,5 +267,141 @@
...
+            {"label": null},
+            {"label": null},
+            {"label": "4", "matrix": [3, 18]},
+            {"label": "5", "matrix": [3, 19]},
+            {"label": "6", "matrix": [3, 20]},
+            {"label": "+", "disabled": true}
+        ],
+        "row4": [
+            {"label": "M4", "matrix": [4, 0]},
+            {"label": "\u21e7", "width": 80, "matrix": [4, 1]},
```
Comment (z3ntu) [inline data/matrix_layouts/razerdefault22.json]: `shift`?

File: data/matrix_layouts/razerdefault22.json [hunk 4]
```diff
@@ -267,5 +267,141 @@
...
+            {"label": "X", "matrix": [4, 4]},
+            {"label": "C", "matrix": [4, 5]},
+            {"label": "V", "matrix": [4, 6]},
+            {"label": "B", "matrix": [4, 7]},
+            {"label": "N", "matrix": [4, 8]},
+            {"label": "M", "matrix": [4, 9]},
+            {"label": ",", "matrix": [4, 10]},
+            {"label": ".", "matrix": [4, 11]},
+            {"label": "-", "matrix": [4, 12]},
+            {"label": "\u21e7", "width": 172, "matrix": [4, 14]},
```
Comment (z3ntu) [inline data/matrix_layouts/razerdefault22.json]: `shift`?

File: data/matrix_layouts/razerdefault22.json [hunk 5]
```diff
@@ -267,5 +267,141 @@
...
+            {"label": "\u2191", "matrix": [4, 16]},
+            {"label": null},
+            {"label": "1", "matrix": [4, 18]},
+            {"label": "2", "matrix": [4, 19]},
+            {"label": "3", "matrix": [4, 20]},
+            {"label": "enter", "height": 126, "matrix": [4, 21]}
+        ],
+        "row5": [
+            {"label": "M5", "matrix": [5, 0]},
+            {"label": "strg", "width": 94, "matrix": [5, 1]},
```
Comment (z3ntu) [inline data/matrix_layouts/razerdefault22.json]: `ctrl`?

File: data/matrix_layouts/razerdefault22.json [hunk 6]
```diff
@@ -267,5 +267,141 @@
...
+            {"label": "\ud83d\udc27", "matrix": [5, 2]},
+            {"label": "alt", "width": 94, "matrix": [5, 3]},
+            {"label": "space", "width": 388},
+            {"label": "alt gr", "width": 94, "matrix": [5, 11]},
+            {"label": "fn"},
+            {"label": "\u2630", "matrix": [5, 13]},
+            {"label": "strg", "width": 92, "matrix": [5, 14]},
```
Comment (z3ntu) [inline data/matrix_layouts/razerdefault22.json]: `ctrl`?

File: data/matrix_layouts/razerdefault22.json (hunk 3 above)
Comment (ananace) [reply data/matrix_layouts/razerdefault22.json]: That's probably more correct, yeah. I just copied the shift-key glyph that was used by other keyboards, to keep it consistent in that manner, not consistent with the physical print on the keyboard.

**Comment by ananace** [comment]
And here's a picture of my own - rather dirty - nordic-layout keyboard, which I was basing this PR off of; https://user-images.githubusercontent.com/534460/65536459-ba991000-df03-11e9-8ff7-585d3878f0e6.jpg

The problem is that the nordic layout has the same three keys, but applies different meanings to them depending on the subregion. As I didn't see any way to set multiple glyphs on the same button, I decided to go with the keyboard I had in front of me, to be able to compare things.  
Though I did copy basically the entire layout from another keyboard, so I should clean up the <kbd>shift</kbd> and <kbd>ctrl</kbd> key definitions, which were different on the one I copied from

**Comment by z3ntu** [comment]
Hmm even the different "Nordic" keyboards look different... Then I'll leave it to your preference :)

**Comment by ananace** [comment]
Unless you see something else that looks off, I'd say go right on ahead

**Comment by z3ntu** [comment]
Row 2 the tab key should be `tab` and not the downwards arrow

**Comment by z3ntu** [comment]
Also cherry-picked this into master. Thanks again!

**Pull request merged**